knack_load_fees.py

```diff
@@ -190,17 +147,27 @@ def main():
     logger.info(f"{len(delete_rows_knack)} to delete")
 
     for row in new_rows_knack:
-        app.record(method="create", obj=KNACK_OBJECT, data=row)
+        try:
+            app.record(method="create", obj=KNACK_OBJECT, data=row)
+        except requests.exceptions.HTTPError as e:
+            logger.error(e.response.text)
+            raise e
```

Comment:
following the knack docs
https://cityofaustin.github.io/knackpy/docs/user-guide/#exceptions